.travis.yml

```diff
@@ -29,9 +26,6 @@ matrix:
     -
       env: PUPPET_GEM_VERSION="~> 5.0" CHECK=parallel_spec
       rvm: 2.4.4
-    -
-      env: PUPPET_GEM_VERSION="~> 4.0" CHECK=parallel_spec RUBYGEMS_VERSION=2.7.8 BUNDLER_VERSION=1.17.3
-      rvm: 2.1.9
```

Comment:
Do we want to drop testing against puppet4?

Reply:
for this module? Heck yeah I assume that only came along due to PDK defaults in the first place, but Core did that work originally.

Reply:
I'm fairly sure that came along with pdk defaults. Otherwise, there'd be an entry for testing with puppet4 in the sync file